app/models/profile.rb

```diff
@@ -62,6 +63,7 @@ def deactivate_for_fraud_review
 
   def reject_for_fraud
     update!(active: false, fraud_review_pending: false, fraud_rejection: true)
+    track_for_attempts_api(:rejected)
```

Comment:
😬 is there a way we can move this to the call sites? I don't love that DB models are making these kinds of connections

Reply:
I had actually initially started to put this in the rake task, but for some reason had it in my head that we had said we didn't want to do that. (I do not recall why I thought that.)

The other part of my thinking here doesn't actually stand up to scrutiny, which was that if we want something akin to an audit trail (kind of what events are), that should happen where the action is, rather than calling `reject_for_fraud` and then separately recording an event after we call it. The counterpoint to this is that virtually all our code does not do this. 😆

Reply:
yeah this class already has a lot of notification stuff built in :[ might as well leave it